tensorboard/plugins/projector/vz_projector/vz-projector-dashboard.ts

```diff
@@ -31,7 +31,7 @@ class VzProjectorDashboard extends PolymerElement {
 
           <li>
             You are not saving any checkpoint. To save your model, create a
-            <a href="https://www.tensorflow.org/api_docs/python/tf/train/Saver"
+            <a href="https://www.tensorflow.org/api_docs/python/tf/compat/v1/train/Saver"
```

Comment:
Running prettier gives me the following formatting changes. Do you mind updating it?

```
           <li>
             You are not saving any checkpoint. To save your model, create a
-            <a href="https://www.tensorflow.org/api_docs/python/tf/compat/v1/train/Saver"
+            <a
+              href="https://www.tensorflow.org/api_docs/python/tf/compat/v1/train/Saver"
               ><code>tf.train.Saver</code></a
             >
             and save your model periodically by calling
```

Reply:
```
(base) ➜  tensorboard git:(broken-link-projector-dashboard) ✗ npm run lint

> tensorboard@0.0.0-unused lint
> prettier --check 'tensorboard/**/*.'{css,html,js,ts,scss} .github/**/*.yml

Checking formatting...
All matched files use Prettier code style!
```
Done, apologies for forgetting to run the check, and thanks for pointing that out.